src/components/ComputeModal.js

```diff
@@ -1516,7 +1530,11 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
               div({ style: { fontSize: 16, fontWeight: 600 } }, ['Use default environment']),
               ul({ style: { paddingLeft: '1rem', marginBottom: 0, lineHeight: 1.5 } }, [
                 li([
-                  div([packageLabel]),
+                  div([packageLabel, h(ClipboardButton, {
+                    text: selectedLeoImage,
+                    style: { marginLeft: '0.5rem' },
+                    tooltip: copyImageLinkTooltipText
```

Comment:
We could just inline the text here:
```suggestion
                    tooltip: 'Copy the image version'
```